### ISSUE TITLE
Plugin compatibility with WKWebView?

### ISSUE THREAD

**Issue by joaolori** [EddyVerbruggen/SocialSharing-PhoneGap-Plugin#1098]
Hello,

Is this plugin compatible with the new WKWebView webview engine?

Thanks

**Comment by tlacroix** [comment]
Hi @joaolori, 

I was asking myself the same question, and I found the answer below.
https://github.com/EddyVerbruggen/SocialSharing-PhoneGap-Plugin/issues/1052

Short: it's been removed in 5.6.0+. If you look at the code, you will see that the block is commented out. Pretty sure that's enough for ITMS-90809.

```
- (NSString*)getIPadPopupCoordinates {
  // see https://github.com/EddyVerbruggen/SocialSharing-PhoneGap-Plugin/issues/1052
  return nil;
  /*
  if (_popupCoordinates != nil) {
    return _popupCoordinates;
  }
  if ([self.webView respondsToSelector:@selector(stringByEvaluatingJavaScriptFromString:)]) {
    return [(UIWebView*)self.webView stringByEvaluatingJavaScriptFromString:@"window.plugins.socialsharing.iPadPopupCoordinates();"];
  } else {
    // prolly a wkwebview, ignoring for now
    return nil;
  }
  */
}
```

**Comment by HansKrywaa** [comment]
I use this Plugin and have no Warning about ITMS-90809

**Comment by tanzanman** [comment]
Yes! I can confirm successful submission of an app today using this plugin at version 5.6.8. App uses WKWebview only along with this plugin and passed. It's really frustrating now that Apple no longer accepts app submissions if their code scanners sniff ITMS-90809 and you have to upload the binary to find that out.  Thanks Eddy for your contributions here!
